### PR TITLE
append log

### PR DESCRIPTION
**Pull request by zhangdexin**

### CODE REVIEW

File: tests/log_manager_test.cc [hunk 1]
```diff
@@ -0,0 +1,114 @@
+//
+// Created by dxz on 2021/7/24.
```
Comment (jovany-wang) [inline tests/log_manager_test.cc:2]: Remove these lines.

File: tests/util.h [hunk 2]
```diff
@@ -223,6 +223,7 @@ class ProxyNode : public raftcpp::rpc::NodeService,
      * We may see the leader as source node forever?
      */
     void HandleRequestPushLogs(RpcConn conn, int64_t committed_log_index,
+                               int32_t pre_log_term_num,
```
Comment (jovany-wang) [inline tests/util.h:226]: Should pre_log_term_num be pre_log_term_index?

**Review by jovany-wang — COMMENTED** [review]
Thanks for this contribution. Looks good overall.

File: tests/log_manager_test.cc (hunk 1 above)
Comment (zhangdexin) [reply tests/log_manager_test.cc:2]: okey

File: tests/util.h (hunk 2 above)
Comment (zhangdexin) [reply tests/util.h:226]: okey

**Review by jovany-wang — APPROVED** [review]
LGTM

**Pull request merged**